src/include/dhcp-server/widgets.rb

```diff
@@ -7,7 +7,9 @@
 # Authors:	Jiri Srain <jsrain@suse.cz>
 
 require "yast"
-require "ui/service_status"
+require "yast2/system_service"
```

Comment:
Please add a dependency to the RPM spec file on the yast2.rpm version that provides these new APIs